src/transformers/models/depth_anything/modeling_depth_anything.py

```diff
@@ -365,9 +365,7 @@ class DepthAnythingForDepthEstimation(DepthAnythingPreTrainedModel):
     def __init__(self, config):
         super().__init__(config)
 
-        self.backbone = AutoBackbone.from_config(
-            config.backbone_config, attn_implementation=config._attn_implementation
-        )
+        self.backbone = load_backbone(config)
```

Comment:
All backbones should be loaded through `load_backbone`. Being able to propogate `attn_implementation` should be done in a follow up (and wasn't being used for depth anything)